File: src/Arrays/javamom/GFG2.java
package Arrays.javamom;

public class GFG2 {

   public static void findElements(int arr[]){
       int n = arr.length;

       int smallest = Integer.MAX_VALUE;
       int secondlargest = Integer.MIN_VALUE;
       int largest = Integer.MIN_VALUE;

       for(int i = 0 ; i<n ; i++){
           if(arr[i] > largest){
               secondlargest = largest;
               largest = arr[i];
           }
           else if(arr[i] > secondlargest && arr[i] != largest){

               secondlargest = arr[i];
           }
           if(arr[i] < smallest){
               smallest = arr[i];
           }
       }
       System.out.println("Smallest" + " : " + smallest);
       System.out.println("Largest" + " : " + largest );
       System.out.println("Secoundlargest" + " : " + secondlargest);
   }

   public static  void main(String [] args){

       int arr [] =  {12 , 34 , 45 , 56, 78 , 1};

       findElements(arr);


   }
}

File: src/Arrays/kotlinbaby/GfG2.kt
package Arrays.kotlinbaby

class GfG2 {

    fun findele(arr : IntArray){

        val n = arr.size

        var smallest = Integer.MAX_VALUE
        var largest = Integer.MIN_VALUE
        var secoundlargest = Integer.MIN_VALUE

        for(i in 0 until arr.size) {
            if(arr[i]>largest){
                secoundlargest = largest
                largest = arr[i]
            }
            else if(arr[i] > secoundlargest && arr[i] != largest){
                secoundlargest = arr[i]
            }
            if (arr[i] < smallest){
                smallest = arr[i]
            }
        }
        println("Smallest : $smallest" )
        println("Largest :  $largest")
        println("Secoundlargest : $secoundlargest")
    }
}